docs/runtime-contract.md

```diff
@@ -181,51 +157,39 @@ for purposes of scaling CPU and removing idle containers.
 
 #### Protocols and Ports
```

Comment:
LGTM 👍 

Summary: `http1` and `h2c` are supported, `http1` by default. No automatic upgrades.